webapp/src/main/webapp/js/webconferencing-jitsi.js

```diff
@@ -281,12 +281,10 @@
             } else {
               process.reject(err);
               log.error("Failed to get call info: " + callId, err);
-              webConferencing.showError("Getting call error", webConferencing.errorText(err));
             }
           } else {
             process.reject();
             log.error("Failed to get call info: " + callId);
-            webConferencing.showError("Getting call error", "Error read call information from the server");
```

Comment:
Could you provide more info in the logged error ? call ID , call title etc ...

Reply:
Ok, i added the available info in the context